components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java

```diff
@@ -442,8 +443,9 @@ public void updateRoleName(int tenantId, String oldRoleName, String newRoleName)
                     while (rSet.next()) {
                         String roleName = rSet.getString(1);
                         if (StringUtils.isNotEmpty(roleName)) {
-                                // Remove the primary domain name from roleNames.
-                                roleList.add(removePrimaryDomainName(roleName));
+
```

Comment:
No need to have a new line.

Reply:
Fixed: 7f95a33846b4f8b080987cf395121938ee0c6ceb